tests/test_roundtrip.rs

```diff
@@ -22,9 +22,10 @@ use std::thread;
 /// a `CHANCES_TO_SKIP` probability of being skipped.
 const CHANCES_TO_SKIP: f64 = 0.9;
 
-const PATHS: [&'static str; 2] = [
+const PATHS: [&'static str; 3] = [
```

Comment:
I guess it could be easier to use `const PATHS: &[&str] = &[...]` here so that we wouldn't have to update the number anymore, but it's fine as-is.